docker/Makefile

```diff
@@ -6,11 +6,15 @@ hadolint_available := $(shell hadolint --help > /dev/null 2>&1; echo $$?)
 hadolint_command := hadolint --ignore DL3008 --ignore DL3018 --ignore DL4000 --ignore DL4001
 hadolint_container := hadolint/hadolint:latest
 pwd := $(shell pwd)
-export BUNDLE_PATH = $(pwd)/.bundle/gems
-export BUNDLE_BIN = $(pwd)/.bundle/bin
-export GEMFILE = $(pwd)/Gemfile
 
-version = $(shell echo $(git_describe) | sed 's/-.*//')
+# Check if ref is set in environment variable
+ifeq ("${REF}","")
+    version := $(shell echo $(git_describe) | sed 's/-.*//')
+else
+    version:= "${REF}"
```

Comment:
Might be good to at least comment here that `REF` is not a git ref, but must be a tag?